roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -69,13 +69,15 @@ def run(self, tmp=None, task_vars=None):
                         msg=str(e),
                     )
 
+            if check.changed:
+                r["changed"] = True
```

Comment:
Right. We can omit from checks, but always set it in the task/action plugin level at most once.

Now whether to always include it on the task/action plugin level will depend on how Ansible interpret it when it is missing (default True or default False) ;-)